frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -359,6 +370,7 @@ export const LoadingDataTableComponent = memo(
       searchQuery,
       useDeepCompareMemoize(props.fieldTypes),
       props.data,
+      props.cellStyles, // maybe, not quite sure...
```

Comment:
i think its not needed, and should be handled by `cellStyles={data?.cellStyles ?? props.cellStyles}` below